source/developers-guide/digital-publishing-elements/index.md

```diff
@@ -357,10 +357,113 @@ public function onContentBannerFilter(\Enlight_Event_EventArgs $args)
 
 Via the event `$args` you can get the return value with `$args->getReturn()`. The return value contains the hole banner data with all layers and elements. You can do some data manipulation and return the data afterwards. For example, you could load some media data, if the user can select media files in your element.
 
+## Data handling with plugin version >= 1.2.0 ##
+With the SwagDigitalPublishing version 1.2.0 a new way of handling your custom elements and their data was implemented.
+<br />
+Each element has to be handled by a so-called `ElementHandler` now.
+<br />
+Those ElementHandlers come in handy when you're trying to add additional data to your own element without having to manipulate the whole banner.
```

Comment:
to provide additional data to your own element without manipulating the original banner.